src/resources/CustomerApi.ts

```diff
@@ -9,9 +9,8 @@ export default class CustomerApi extends Base {
       .then(response => new Credit(response.data))
   }
 
-  async priceList (): Promise<Price> {
-    return this.axios.get('/customers/' + this.customer + '/pricelist')
+  async priceList (currency?: 'USD|EUR'): Promise<Price> {
+    return this.axios.get('/customers/' + this.customer + '/pricelist', { params: currency ? { currency } : undefined })
```

Comment:
It should be `currency?: 'USD' | 'EUR'` otherwise it would expect to be called like `api.customer.priceList('USD|EUR')`.